exonum/src/blockchain/tests.rs

```diff
@@ -622,3 +527,23 @@ mod rocksdb_tests {
         super::assert_service_execute(&blockchain, &mut db);
     }
 }
+
+#[derive(Debug, ProtobufConvert)]
+#[exonum(pb = "protobuf::tests::TestProtobufConvert", crate = "crate")]
+struct TestProtobufConvertTypes {
+    key: PublicKey,
+    hash: Hash,
+    unsigned_32: u32,
+    unsigned_64: u64,
+    regular_i32: i32,
+    regular_i64: i64,
+    fixed_u32: u32,
+    fixed_u64: u64,
+    fixed_i32: i32,
+    fixed_i64: i64,
+    float_32: f32,
+    float_64: f64,
+    boolean: bool,
+    s_i32: i32,
+    s_i64: i64,
+}
```

Comment:
Can you write encode/decode tests? And I think that blockchain mod is not good place for this test.
Maybe we need to revamp modules structure.

Reply:
Exactly I am tried changing mod struct - since protobuf is the only encoding now + tests are planned to.